scipy/optimize/tests/test_linprog.py

```diff
@@ -1675,6 +1675,10 @@ def test_crossover(self):
         res = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq,
                       bounds=bounds, method=self.method, options=self.options)
         # there should be nonzero crossover iterations for IPM (only)
+        # TODO: highs counts crossover iterations differently, so this test
+        #       now fails
+        if self.method == "highs-ipm":
+            pytest.skip("HiGHS crossover iteration behavior changed!")
```

Comment:
> So maybe this is just too simple of a problem and crossover is no longer needed?

I think this is correct.  The cross-over has "primal/dual" pushes which used to be counted as cross-over iterations, but has been changed to the number of basis updates during crossover, which for this problem is 0.

> Rather than skipping the test, maybe we need to change the problem so that res.crossover_nit is nonzero for highs-ipm.

I'm not sure how to come up with such a pathological crossover test -- do we just have to guess the right problem here?

Reply:
It's much more common than one might think. I checked and I get a `crossover_nit` with `magic_square(4)`.

Reply:
Great!  I'll switch the test to use that problem

Reply:
see mckib2/scipy#30